### PR TITLE
Feature protected routes

### PR DESCRIPTION
**Pull request by Meggie-H**

### CODE REVIEW

File: src/components/BuildGraph.tsx [hunk 1]
```diff
@@ -50,11 +49,9 @@ const BuildGraph = () => {
   }
 
   return (
-    <div className="flex flex-col items-center rounded-2xl bg-gray-900 p-4">
-      <h2 className="text-gray-200">Builds</h2>
+    <div className="flex flex-col items-center border border-gray-800 rounded-2xl p-4">
+      <h2 className="width-full text-gray-200">Builds</h2>
```
Comment (yanevdw) [inline src/components/BuildGraph.tsx]: Should this not be `w-full`?

File: src/components/CommitGraph.tsx [hunk 2]
```diff
@@ -62,13 +63,11 @@ const CommitGraph = () => {
   }
 
   return (
-    <div className="flex h-full flex-col items-center rounded-2xl bg-gray-900 p-4">
-      <h2 className="text-gray-200">Commits</h2>
+    <div className="flex h-full flex-col items-center border border-gray-800 p-4">
+      <h2 className="width-full text-gray-200">Commits</h2>
```
Comment (yanevdw) [inline src/components/CommitGraph.tsx]: `w-full` here too.

File: src/components/LanguageGraph.tsx [hunk 3]
```diff
@@ -45,11 +44,9 @@ const LanguageGraph = () => {
   }
 
   return (
-    <div className="flex flex-col items-center rounded-2xl bg-gray-900 p-4">
-      <h2 className="text-gray-200">Languages</h2>
+    <div className="flex flex-col items-center border border-gray-800 p-4">
+      <h2 className="width-full text-gray-200">Languages</h2>
```
Comment (yanevdw) [inline src/components/LanguageGraph.tsx]: `w-full` here too, just search through all your files and update it.

**Review by yanevdw — APPROVED** [review]
Fantastic work! 

Some things to note:
1. Just check for any width-full in your styling, these should be `w-full`.
2. If you do not end up implementing the GitGraph, please remember to remove it.
3. Consider destructuring the useQuery for example 
```const {
    data,
    error,
    isLoading,
    isPending,
  } = useQuery({
   ---
  }); 
  ```
  This is up to your preference.

**Review by yanevdw — APPROVED** [review]
Tailwind fixes have been made. Well done!

File: src/components/CommitGraph.tsx
Comment (LesediMoko) [inline src/components/CommitGraph.tsx:1]: Consider separating out your chart data and options into a single config-like file and then import then? To pass in the contributors and contributorNames you can setup a function that takes them in spits out chartData

File: src/components/IssueGraph.tsx
Comment (LesediMoko) [inline src/components/IssueGraph.tsx:1]: Don't forget to handle isPending/isLoading and isError from your useQuery

**Review by LesediMoko — CHANGES_REQUESTED** [review]
Looks great, Meg! Just missing some error handling and a few minor things regarding separating concerns. All in all, lovely work!

File: src/components/IssueGraph.tsx
Comment (Meggie-H) [reply src/components/IssueGraph.tsx:1]: In Issue graph? I do have it in Issue graph

File: src/components/CommitGraph.tsx
Comment (Meggie-H) [reply src/components/CommitGraph.tsx:1]: Oh I understand, I prefer it this way as the charts are not so similar as they seem, so I prefer to be able to change the way they look separately

File: src/components/IssueGraph.tsx
Comment (LesediMoko) [reply src/components/IssueGraph.tsx:1]: Oh my bad, I forgot to expand the file. My bad, Meg 😭

**Review by LesediMoko — APPROVED** [review]
Great stuff! Well done on all your hard work 👏

File: src/components/CommitGraph.tsx
Comment (LesediMoko) [reply src/components/CommitGraph.tsx:1]: Makes complete sense. Keep as it is ✨

**Pull request merged**